zummon/src/commonMain/kotlin/companysurvey/GridConnection.kt

```diff
@@ -17,7 +17,7 @@ data class GridConnection(
     // Is always set when object comes from the database.
     val sequence: Int? = null,
 
-    val electricity: Electricity = Electricity(),
+    val electricity: Electricity? = Electricity(),
```

Comment:
Better keep it non-nullable because in AnyLogic/Java there is no null-safe navigation operator "?." so the modellers will have to write extra null checks.